From 18746c99714dd04297fb3968b6c84f4f718d20bd Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Mon, 2 May 2011 10:51:11 +0200
Subject: [PATCH 4/4] e-wm-config-illume2-shr: drop conf_transitions,
 conf_fonts, conf_startup, conf_wallpaper, conf_scale,
 those were merged with conf_theme in r59059

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 e-wm/e-wm-config-illume2-shr/e.src |   30 ------------------------------
 1 files changed, 0 insertions(+), 30 deletions(-)

diff --git a/e-wm/e-wm-config-illume2-shr/e.src b/e-wm/e-wm-config-illume2-shr/e.src
index 0c79807..2cbf7df 100644
--- a/e-wm/e-wm-config-illume2-shr/e.src
+++ b/e-wm/e-wm-config-illume2-shr/e.src
@@ -91,12 +91,6 @@ group "E_Config" struct {
 			value "priority" int: 0;
 		}
 		group "E_Config_Module" struct {
-			value "name" string: "conf_transitions";
-			value "enabled" uchar: 1;
-			value "delayed" uchar: 1;
-			value "priority" int: 0;
-		}
-		group "E_Config_Module" struct {
 			value "name" string: "conf_display";
 			value "enabled" uchar: 1;
 			value "delayed" uchar: 1;
@@ -109,36 +103,12 @@ group "E_Config" struct {
 			value "priority" int: 0;
 		}
 		group "E_Config_Module" struct {
-			value "name" string: "conf_fonts";
-			value "enabled" uchar: 1;
-			value "delayed" uchar: 1;
-			value "priority" int: 0;
-		}
-		group "E_Config_Module" struct {
-			value "name" string: "conf_startup";
-			value "enabled" uchar: 1;
-			value "delayed" uchar: 1;
-			value "priority" int: 0;
-		}
-		group "E_Config_Module" struct {
 			value "name" string: "conf_theme";
 			value "enabled" uchar: 1;
 			value "delayed" uchar: 1;
 			value "priority" int: 0;
 		}
 		group "E_Config_Module" struct {
-			value "name" string: "conf_wallpaper";
-			value "enabled" uchar: 1;
-			value "delayed" uchar: 1;
-			value "priority" int: 0;
-		}
-		group "E_Config_Module" struct {
-			value "name" string: "conf_scale";
-			value "enabled" uchar: 1;
-			value "delayed" uchar: 1;
-			value "priority" int: 0;
-		}
-		group "E_Config_Module" struct {
 			value "name" string: "conf_interaction";
 			value "enabled" uchar: 1;
 			value "delayed" uchar: 1;
-- 
1.7.5.rc3

